<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regDialog</class>
 <widget class="QDialog" name="regDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>345</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QLabel" name="labelAccount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>卡号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBank">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>银行：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelId">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>持卡人身份证号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelContact">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>持卡人联系方式：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNum">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditId">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditContact">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxbank">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSubmit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>12</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Store/icon.qrc">:/Icon/Icon/useradd-white-48.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>labelAccount</zorder>
  <zorder>labelPassword</zorder>
  <zorder>labelBank</zorder>
  <zorder>labelId</zorder>
  <zorder>labelContact</zorder>
  <zorder>lineEditNum</zorder>
  <zorder>lineEditPassword</zorder>
  <zorder>lineEditId</zorder>
  <zorder>lineEditContact</zorder>
  <zorder>comboBoxbank</zorder>
  <zorder>buttonSubmit</zorder>
  <zorder>buttonClear</zorder>
  <zorder>buttonCancel</zorder>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditNum</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>comboBoxbank</tabstop>
  <tabstop>lineEditId</tabstop>
  <tabstop>lineEditContact</tabstop>
  <tabstop>buttonSubmit</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonClear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
